clients/drcachesim/tracer/func_trace.cpp

```diff
@@ -271,6 +279,12 @@ func_trace_init(func_trace_append_entry_t append_entry_,
         DR_ASSERT(false);
         goto failed;
     }
+
+    drwrap_set_global_flags(DRWRAP_NO_FRILLS);
+    drwrap_set_global_flags(DRWRAP_FAST_CLEANCALLS);
+    drwrap_set_global_flags(DRWRAP_SAFE_READ_RETADDR);
```

Comment:
The safe-read flags do add a little overhead, but are a good idea.  Maybe mention this in the commit msg since you're mentioning the perf flags.